hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -1,40 +1,15 @@
-# Pi 4 Bootloader Configuration
+# Raspberry Pi 4 Bootloader Configuration
```

Comment:
Style guide (https://github.com/raspberrypilearning/style-guide/blob/master/style-guide.md) says only first word (and product names) should be capitalised, so this becomes:

`Raspberry Pi 4 bootloader configuration`